cpp/installer.cc

```diff
@@ -464,6 +464,12 @@ void InstallUBTools(const bool make_install, const OSSystemType os_system_type)
         ExecUtil::ExecOrDie(ExecUtil::LocateOrDie("mkdir"), { "-p", UBTools::GetTuelibPath() });
     }
 
+    // ..and /usr/local/var/log/tuefind
+    if (not FileUtil::Exists(UBTools::GetTuefindLogPath())) {
+        Echo("creating " + UBTools::GetTuefindLogPath());
+        ExecUtil::ExecOrDie(ExecUtil::LocateOrDie("mkdir"), { "-p", UBTools::GetTuefindLogPath() });
```

Comment:
Please use FileUtil::MakeDirectory instead of shelling out.  In general, please prefer our functions over shelling out.